exotica_core/src/planning_problem.cpp

```diff
@@ -294,6 +294,11 @@ std::pair<std::vector<double>, std::vector<double>> PlanningProblem::GetCostEvol
     return ret;
 }
 
+int PlanningProblem::GetNumberOfIterations() const
+{
+    return std::get<0>(GetCostEvolution()).size();
```

Comment:
Size is size_t, you have an implicit cast to int

Reply:
So should this line be `std::get<0>(GetCostEvolution()).size_t();`? Sorry, I am not so versed with C++.

Reply:
`static_cast<int>(std::get<0>(GetCostEvolution()).size());`